lib/unifile-dropbox.js

```diff
@@ -11,6 +11,8 @@ const {UnifileError, BatchError} = require('./error');
 const NAME = 'dropbox';
 const DB_OAUTH_URL = 'https://www.dropbox.com/oauth2';
 
+const charsToEncode = /[\u007f-\uffff]/g;
```

Comment:
\u007f is char delete? Here they [seem to say it is the wrong regexp to match latin1](https://stackoverflow.com/questions/31039531/u007f-and-uffff-in-js-regexp), it may be interesting to you?
Also maybe a comment here would help :)
It is not blocking, just a question

Reply:
I don't know, I got this method from Dropbox support so I just past it confidently 😃